src/core/image/image.service.test.ts

```diff
@@ -0,0 +1,69 @@
+import { ImageService } from './image.service';
+import { getTestImage, getTestImageBuffer, streamToBuffer } from '../../../test/test.utils';
+import type { Region } from 'sharp';
+import sharp from 'sharp';
+
+import getColors from 'get-image-colors';
+
+
+describe('ImageService', () => {
+  let imageService: ImageService;
+  beforeEach(() => {
+    imageService = new ImageService();
+  });
+
+  describe('#resize', () => {
+    it('should resize passed image', async () => {
+      const expectedDimensions = 20;
+      const image = await getTestImage();
+
+      const resizedImage = imageService.resize(image, {height: expectedDimensions});
+
+      const resizedImageMetadata = await sharp(await streamToBuffer(resizedImage)).metadata();
```

Comment:
I would prefer to create function like `getImageSize` because when we decide sharp is useless and we gonna use another library then modifying this function is enough.